continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -61,6 +61,9 @@ test:
     - dask-scheduler --help
     - dask-ssh --help
     - dask-worker --help
+    - dask scheduler --help
+    - dask ssh --help
+    - dask worker --help
```

Comment:
For ease of comparison, could we please line these up to match [the listed order in the feedstock]( https://github.com/conda-forge/distributed-feedstock/blob/c69dfd4b07dda15f51b6aa18bb8826a2d7e72341/recipe/meta.yaml#L58-L63 )?

Reply:
Might be missing something here, but it looks like the order does match up the conda-forge feedstock?

Reply:
Oops you're right. I misread. Sorry about that 🤦‍♂️